src/Jira/Api.php

```diff
@@ -82,15 +81,18 @@ public function __construct(
         $this->client = $client;
     }
 
+    /**
+     * @param int $options
```

Comment:
Not sure if short description in mandatory in DocBlock but I usually always add it.

Same is true for other DocBlocks.